python/lsst/sims/maf/metricBundles/metricBundleGroup.py

```diff
@@ -345,7 +357,12 @@ def plotAll(self, savefig=True, outfileSuffix=None, figformat='pdf', dpi=600, th
         plotHandler = PlotHandler(outDir=self.outDir, resultsDb=self.resultsDb,
                                   savefig=savefig, figformat=figformat, dpi=dpi, thumbnail=thumbnail)
         for b in self.bundleDict.itervalues():
-            b.plot(plotHandler=plotHandler, outfileSuffix=outfileSuffix, savefig=savefig)
+            # Check that there are metric values to plot
+            if b.metricValues is not None:
```

Comment:
Shouldn't this be taken care of in the plotHandler, rather than here? 
(is this duplicated code?)